packages/devextreme/testing/testcafe/helpers/shadowDom/shadowDomExtension.js

```diff
@@ -7,6 +7,22 @@ function get(selector) {
     return typeof selector === 'string' && selector ? getRoot()?.querySelectorAll(selector) : selector;
 }
 
+window.addShadowRootTree = () => {
+    const root = document.querySelector('#parentContainer');
+    const childNodes = root.childNodes;
+
+    if(root.shadowRoot) {
+        return;
+    }
+
+    root.attachShadow({ mode: 'open' });
+
+    const shadowContainer = document.createElement('div');
+    shadowContainer.append.apply(shadowContainer, Array.from(childNodes));
```

Comment:
minor
```suggestion
    shadowContainer.append(...childNodes);
```

Reply:
discussed, there's some error with this construction, will refactor in future for all places like this